src/components/Nav.tsx

```diff
@@ -15,6 +16,9 @@ const Navbar: React.FC = () => {
         <Link href="/" className="uppercase">
           Trend Track
         </Link>
+        <Link href="https://github.com/Bashamega/TrendTrack" target="_blank">
```

Comment:
make it near the api text. You can achieve this by wrapping the github icon and the api link in one div

Reply:
done